mava/types.py

```diff
@@ -124,6 +149,7 @@ class Observation(NamedTuple):
     agents_view: chex.Array  # (num_agents, num_obs_features)
     action_mask: chex.Array  # (num_agents, num_actions)
     step_count: Optional[chex.Array] = None  # (num_agents, )
+    graph: Optional[GraphsTuple] = None
```

Comment:
Instead of adding this could we rather make a new type `GraphObservation` which looks like this:

```python
class GraphObservation(NamedTuple):
    observation: Observation | ObservationGlobalState
    graph: GraphsTuple
```

Please double check that this actually will work - I think it should as you should be able to handle this new in the network and your wrappers can return this type.